build.xml

```diff
@@ -439,6 +439,11 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
         <ivy:cachepath pathid="owasp-classpath" conf="owasp"/>
     </target>
 
+    <target name="report" depends="init,ivy-init">
```

Comment:
Why is the target called "report" when it's the "dependency-report"? Wouldn't that make more sense? Or "ivy-dependency-report"? (like that one better tbh)

Reply:
ps. doesn't apply cleanly to 3.4 (would be nice to have there as well)

Reply:
Fixed and fixed.

https://github.com/apache/zookeeper/pull/398